rpcs3/Emu/System.h

```diff
@@ -146,6 +173,7 @@ class Emulator final
 
 	void Init(bool add_only = false);
 
+	std::unique_ptr<utils::serial> ar;
```

Comment:
I was confused without the member prefix
```suggestion
	std::unique_ptr<utils::serial> m_ar;
```

Reply:
But it's a public member.

Reply:
I can probably make it private with caccess function returning raw pointer.

Reply:
Fixed.